### PR TITLE
always display full list of options

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-13 at 4 43 05 pm

File: src/languageProvider/locales/en_US.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
   "oracle.consensus": "Consensus Threshold {value}. This value indicates the amount of BOT needed to set the result.",
   "oracle.enterAmountDisabledText": "Please entered a valid amount.",
   "oracle.eventUnconfirmed": "This created Event is unconfirmed. You cannot interact with it until it is confirmed by the blockchain.",
+  "oracle.optionIsPrevResult": "This option was set as result in previous rounds",
```
Comment (dwalintukan) [inline src/languageProvider/locales/en_US.json]: "This option was set as the result in the previous round"
- added "the"
- use singular round, not rounds. as we only disable the result of the previous round.

File: src/scenes/Event/components/EventOption/index.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ import Typography from 'material-ui/Typography';
 import classNames from 'classnames';
 import { LinearProgress } from 'material-ui/Progress';
 import { withStyles } from 'material-ui/styles';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
```
Comment (dwalintukan) [inline src/scenes/Event/components/EventOption/index.js]: remove intlShape

File: src/scenes/Event/components/EventOption/index.js [hunk 3]
```diff
@@ -71,7 +73,12 @@ class EventOption extends React.PureComponent {
                   <div className={classes.eventOptionProgressNum}>{percent}%</div>
                 </div>
                 <Typography variant="body1">
-                  {amount}
+                  {isPrevResult ?
+                    <FormattedMessage
+                      id="oracle.optionIsPrevResult"
+                      defaultMessage="This option was set as result in previous rounds"
```
Comment (dwalintukan) [inline src/scenes/Event/components/EventOption/index.js]: change to my comment above

**Pull request merged**